distribution/lib/Standard/Table/0.0.0-dev/src/Data/Conversions/Convertible_To_Rows.enso

```diff
@@ -16,12 +16,20 @@ type Convertible_To_Rows
        - getter: Get the value for a specified row.
     Value length:Integer (getter : Integer->Any)
 
+    ## PRIVATE
+       Return the iterator values as a `Vector`.
+    to_vector : Vector Any
+    to_vector self = 0.up_to self.length . map self.getter
```

Comment:
Note that this given a List of length N will take O(N^2) time, because taking each `i`th index will require iterating thru the list again and again within `list.get`.

Reply:
Using your change.